packages/core/src/index.js

```diff
@@ -3,7 +3,7 @@ export default middleware
 
 export { runSaga } from './internal/runSaga'
 export { END, eventChannel, channel, multicastChannel, stdChannel } from './internal/channel'
-export { CANCEL, SAGA_LOCATION } from './internal/symbols'
+export { CANCEL, SAGA_LOCATION, globalSymbolNames } from './internal/symbols'
```

Comment:
we do not import/require it from the babel plugin (and probably we won't, using global symbol registry is "good enough"), so probably better to just hide it in the internals, probably not even worth to keep it as a map as we have only a single item there

I'd just go with same, duplicated, implementation for SAGA_LOCATION in both core and babel plugin (and wouldnt also export SAGA_LOCATION from here, until somebody requests it)